web/template/layout/page.gohtml

```diff
@@ -19,7 +19,7 @@
       <link href="{{ prefix "/stylesheets/all.css" }}" rel="stylesheet">
     </head>
 
-    <body class="govuk-template__body app-body-class">
+    <body class="govuk-template__body app-body-class" data-prefix="{{ prefix "" }}">
```

Comment:
FWIW, I'd normally put something like this in a meta tag `<meta name="sirius-prefix" content="{{ prefix "" }}">`